src/app/shared/components/profile-image/profile-image.component.html

```diff
@@ -4,7 +4,7 @@
       *ngIf="!profileImageData?.image || profileImageData?.image==''; else Img; let image">
       <img class="thumbnail-image" src="../../../../assets/prof-img/user.png" />
       <div class="camera-icon-container" *ngIf="isMobile">
-        <ion-icon name="camera" class="camera-icon" (click)="uploadPhoto('CAMERA')"></ion-icon>
+        <ion-icon name="camera" class="camera-icon" (click)="uploadPhoto('CAMERA'); clearFileInput();"></ion-icon>
```

Comment:
If u do like this, only after upload function is executed, then only clearFileInput function will get called right? pls confirm that